Floofbot/Services/EventLoggerService.cs

```diff
@@ -595,5 +602,37 @@ public Task UserUnmuted(IUser user, IUser unmuter, IGuild guild)
             });
             return Task.CompletedTask;
         }
+
+        class WordFilterService
+        {
+            List<FilteredWord> _wordfilterList;
+
+            bool hasFilteredWord(FloofDataContext floofDb, string messageContent, ulong serverId, ulong channelId)
+            {
+                // whitelist means we don't have the filter on for this channel
+                if (floofDb.FilterChannelWhitelists.AsQueryable()
+                    .Any(x => x.ChannelId == channelId && x.ServerId == serverId))
```

Comment:
also need to check FilterConfigs either if server has not been set uo (no entry) or disabled on whole server (IsOn = false)